arm-powerbiembedded/2016-01-29/swagger/powerbiembedded.json

```diff
@@ -0,0 +1,1041 @@
+{
+    "swagger": "2.0",
+    "host": "management.azure.com",
+    "info": {
+        "version": "2016-01-29",
+        "title": "Power BI Embedded Management Client",
+        "description": "Azure Resource Provider for Power BI Embedded"
+    },
+    "schemes": [
+        "https",
+        "http"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.PowerBI/workspaceCollections/{workspaceCollectionName}": {
+            "get": {
+                "tags": [
+                    "WorkspaceCollections"
+                ],
+                "operationId": "WorkspaceCollections_getByName",
+                "description": "The **Get Power BI Workspace Collection** operation retrieves an existing Power BI Workspace Collection.",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "name": "subscriptionId",
```

Comment:
subscriptionId should be a global parameter. When defined globally this will be a property on the client and the user will not have to provide subscriptionId in eery operation. Just set it as a one time thing or provide it in the constructor while creating the client.
Please look at the storage spec as an example https://github.com/Azure/azure-rest-api-specs/blob/master/arm-storage/2016-01-01/swagger/storage.json#L978.

ApiVersion is also a global parameter.

The storage spec is a good example for looking at operation group and model definition.
They use StorageAccount**s_** in the operationId and have a model definition named StorageAccount. This is pretty similar to WorkspaceCollections in this spec.